docs/guide/content_rendering/embed_and_list_content/render_images.md

```diff
@@ -67,3 +67,46 @@ To use it, select the variation when rendering the image:
     }
 }) }}
 ```
+
+## Use focal point
+
+In the [image editor](../../image_editor.md) you can define a focal point for an image.
+The focal point doesn't have an instant effect when you use the default templates.
+However, you can use it to select the part of the image the view focuses on when the image is cropped.
+
+The following example shows how to use an image contained in an Image Field as a focussed background.
```

Comment:
I would also mention that there are different approaches to focal point and JS libraries as well.